src/core_modules/capture-core/reducers/descriptions/app.reducerDescriptionGetter.js

```diff
@@ -69,7 +69,7 @@ export const getAppReducerDesc = (appUpdaters: Updaters) => createReducerDescrip
     },
     [enrollmentPageActionTypes.PAGE_OPEN]: (state) => {
         const newState = { ...state };
-        newState.page = null;
+        newState.page = 'enrollment';
```

Comment:
hey @simonadomnisoru , is there a reason why this has changed to "enrollment", I don't see it's used anywhere but seeing other places they are all `null`

Reply:
Hey @jasminenguyennn , 
When I enabled the direct navigation to the enrollment dashboard some browser history bugs appeared that were not visible before. When the enrollment page opens (in this case on `enrollmentPageActionTypes.PAGE_OPEN`), the page should be set to `enrollment` not `null`.  This change fix the previous undetected errors. 
Thanks!